build.sbt

```diff
@@ -0,0 +1,143 @@
+val previousVersion: Option[String] = Some("1.1.0")
+val newScalaBinaryVersionsInThisRelease: Set[String] = Set()
+
+inThisBuild(Def.settings(
+  version := "1.1.1-SNAPSHOT",
+  organization := "org.scala-js",
+  scalaVersion := "2.12.11",
+  crossScalaVersions := Seq("2.11.12", "2.12.11", "2.13.2"),
+  scalacOptions ++= Seq(
+    "-deprecation",
+    "-feature",
+    "-Xfatal-warnings",
+    "-encoding", "utf-8",
+  ),
+
+  // Licensing
+  homepage := Some(url("https://github.com/scala-js/scala-js-js-envs")),
+  startYear := Some(2013),
+  licenses += (("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0"))),
+  scmInfo := Some(ScmInfo(
+      url("https://github.com/scala-js/scala-js-js-envs"),
+      "scm:git:git@github.com:scala-js/scala-js-js-envs.git",
+      Some("scm:git:git@github.com:scala-js/scala-js-js-envs.git"))),
+
+  // Publishing
+  publishMavenStyle := true,
+  publishTo := {
+    val nexus = "https://oss.sonatype.org/"
+    if (version.value.endsWith("-SNAPSHOT"))
+      Some("snapshots" at nexus + "content/repositories/snapshots")
+    else
+      Some("releases" at nexus + "service/local/staging/deploy/maven2")
+  },
+  pomExtra := (
+    <developers>
+      <developer>
+        <id>sjrd</id>
+        <name>Sébastien Doeraene</name>
+        <url>https://github.com/sjrd/</url>
+      </developer>
+      <developer>
+        <id>gzm0</id>
+        <name>Tobias Schlatter</name>
+        <url>https://github.com/gzm0/</url>
+      </developer>
+      <developer>
+        <id>nicolasstucki</id>
+        <name>Nicolas Stucki</name>
+        <url>https://github.com/nicolasstucki/</url>
+      </developer>
+    </developers>
+  ),
+  pomIncludeRepository := { _ => false },
+))
+
+val commonSettings = Def.settings(
+  // Links to the JavaDoc do not work
+  Compile / doc / scalacOptions -= "-Xfatal-warnings",
```

Comment:
Note that I deactivate fatal warnings in Scaladoc, because links to the JavaDoc, including in `@throws` sections, do not work and report errors. We could fix this by doing all the heavy machinery that we do in scala-js/scala-js to patch up the JavaDoc links, but I found that overkill right now.